translator/util/ec2util/ec2util.go

```diff
@@ -24,20 +27,26 @@ type ec2Util struct {
 	AccountID  string
 }
 
-const allowedRetries = 5
+var (
+	ec2UtilInstance *ec2Util
+	once            sync.Once
+)
 
-var e *ec2Util
-var once sync.Once
+const (
+	allowedNetworkRetries = 5
+	allowedIMDSRetries = 2 
```

Comment:
This is duplicated from the constants file. Is the constant from that file used at all?

Reply:
I would not say its duplicated from constants files since it is separated between two package. For the second question, yes. However, the reason I created two constants would be: separate the TimeOut Strategies between Translator and also EC2Tagger. Lets me know if that make sense.